src/test/java/seedu/address/logic/commands/HelpCommandTest.java

```diff
@@ -14,7 +14,29 @@ public class HelpCommandTest {
 
     @Test
     public void execute_help_success() {
-        CommandResult expectedCommandResult = new CommandResult(SHOWING_HELP_MESSAGE, true, false);
+        CommandResult expectedCommandResult = new CommandResult(SHOWING_HELP_MESSAGE,
+                true, false);
         assertCommandSuccess(new HelpCommand(), model, expectedCommandResult, expectedModel);
+
+        CommandResult expectedCommandResultAdd = new CommandResult(AddCommand.MESSAGE_USAGE,
+                false, false);
+        assertCommandSuccess(new HelpCommand(AddCommand.MESSAGE_USAGE),
+                model, expectedCommandResultAdd, expectedModel);
+
+        CommandResult expectedCommandResultClear = new CommandResult(AddCommand.MESSAGE_USAGE,
```

Comment:
Are the expected command results to be checked from here onwards meant to match the corresponding commands in their names?